config/default-amo.js

```diff
@@ -24,6 +24,21 @@ module.exports = {
   },
   enableTrailingSlashesMiddleware: true,
   fxaConfig: 'amo',
+
+  // This needs to be kept in sync with addons-server's SUPPORTED_NONLOCALES
+  // settings value: https://github.com/mozilla/addons-server/blob/master/src/olympia/lib/settings_base.py#L274
+  validRootUrlExceptions: [
```

Comment:
I'm confused. Is this supposed to be `validRootExceptions` or is the whitelisted key supposed to be `validRootUrlExceptions`? Either way, they are out of sync.